src/Tribe/I18n.php

```diff
@@ -190,13 +250,25 @@ public function compile_translations( array $strings, $domains ) {
 
 				// Grab the possible strings for default and any other domain.
 				if ( 'default' === $domain ) {
-					$strings[ $key ][] = __( $value );
-					$strings[ $key ][] = __( strtolower( $value ) );
-					$strings[ $key ][] = __( ucfirst( $value ) );
+					if ( $flags & static::COMPILE_INPUT ) {
+						$strings[ $key ][] = __( $value );
```

Comment:
:no_entry_sign: The $text arg must be a single string literal, not "$value".

<sub>hash: 3ec79062236b134ebbab2ce8bf707925</sub>